iris.cabal

```
@@ -27,7 +28,7 @@ source-repository head
   location:            https://github.com/chshersh/iris.git
 
 common common-options
-  build-depends:       base >= 4.14 && < 4.18
+  build-depends:       base >= 4.14 && < 4.19
```

Let's actually bump the upper bound to `< 5`.
```suggestion
  build-depends:       base >= 4.14 && < 5
```

I actually wrote a blog post about this recently 😅 

* https://dev.to/chshersh/new-oss-rule-do-as-little-work-as-possible-4j2k